hw_c/second_kr/palindrom.c

```diff
@@ -0,0 +1,90 @@
+#include "list.h"
+#include <stdio.h>
+
+
+int palindrom(LinkedList* numbers, int len)
+{
+    int j = len - 1;
+    int isPalindrom = 1;
+    for (int i = 0; i <= len / 2; i++) {
+        int start = getValue(numbers, i);
+        int finish = getValue(numbers, j);
+        if (start != finish) {
+            // printf("NOT palindrom\n");
+            isPalindrom--;
```

Comment:
Никогда так не делайте.

```suggestion
            isPalindrom = 0;
```

А лучше, `#include <stdbool.h>` и

```suggestion
            isPalindrom = false;
```